core/conn/odb/src/odb.c

```diff
@@ -6246,7 +6248,8 @@ static void Oload(int eid)
              trt[16];           /* translit to array */         
         char op;                /* 1=substr, 2=dconv, 3=tconv, 4=tsconv, 5=replace,
                                    6=toupper, 7=tolower, 8=firstup, 9=csubstr, 10=translit,
-                                   11=comp, 12=comp3, 13=zoned, 14=emptyasconst, 15=emptyasempty */
+                                   11=comp, 12=comp3, 13=zoned, 14=emptyasconst, 15=emptyasempty,
+                                   16=div */
```

Comment:
Should the comments also include 17=trim?

Reply:
Thank you, I will fix this.